include/alpaka/core/BoostPredef.hpp

```diff
@@ -12,26 +13,24 @@
 #endif
 
 //---------------------------------------HIP-----------------------------------
-// __HIPCC__ is defined by hipcc (if either __CUDACC__ is defined)
+// __HIPCC__ is defined by hipcc when either __CUDACC__, __HIP__ or __HCC__ are defined.
 // https://github.com/ROCm-Developer-Tools/HIP/blob/master/docs/markdown/hip_porting_guide.md#compiler-defines-summary
 #if !defined(BOOST_LANG_HIP)
-#    if defined(__HIPCC__) && (defined(__CUDACC__) || defined(__HIP__))
-#        include <hip/hip_runtime.h>
```

Comment:
I do not thing cou should remove the include else `HIP_VERSION_MAJOR` few lines later will not be defined.

Reply:
The include is replaced by `<hip/hip_version.h>` a few lines below.